src/pages/match/ImageCard.jsx

```diff
@@ -14,17 +14,17 @@ export default function ImageCard({ titleId, annotation }) {
   const {
     data: siteSettings,
     loading,
-    siteSettingsVersion,
+    dataUpdatedAt: siteSettingsTimestamp,
   } = useSiteSettings();
 
   const regionChoices = useMemo(
     () =>
-      get(
-        siteSettings,
-        ['site.custom.regions', 'value', 'locationID'],
-        [],
-      ),
-    [siteSettingsVersion, siteSettings],
+      get(siteSettings, [
+        'site.custom.regions',
```

Comment:
If you only want the one value, why are you getting the block (or does this use a cache of the block)

Reply:
the cache of the block. it actually should be faster to use the site settings block that was already fetched than to fetch only the value that's needed in a separate request